webrtc.html

```diff
@@ -4641,29 +4648,35 @@ <h2>Methods</h2>
         secure module), a reference to the private key can be held in
         the <a>[[\KeyingMaterial]]</a> internal slot, allowing the
         private key to be stored and used.</p>
-        <p>When a <a>user agent</a> is required to obtain a structured
-        clone [[!HTML51]] of an <code>RTCCertificate</code> object,
-        it performs the following steps:</p>
+        <p><code>RTCCertificate</code> objects are <a>serializable objects</a>
+        [[!HTML]]. Their <a>serialization steps</a>, given <var>value</var> and
+        <var>serialized</var>, are:</p>
+
         <ol>
-          <li>Let <var>input</var> and <var>memory</var> be the corresponding
-          inputs defined by the internal structured cloning algorithm, where
-          <var>input</var> represents an <code>RTCCertificate</code> object to
-          be cloned.</li>
-          <li>Let <var>output</var> be a newly constructed
-          <code>RTCCertificate</code> object.</li>
-          <li>Copy the value of the <code>expires</code> attribute from
-          <var>input</var> to <var>output</var>.</li>
-          <li>Let the <a>[[\Certificate]]</a> internal slot of <var>output</var>
-          be set to the result of invoking the internal structured clone
-          algorithm recursively on the corresponding internal slots of
-          <var>input</var>, with the slot contents as the new
-          "<var>input</var>" argument and <var>memory</var> as the new
-          "<var>memory</var>" argument.
-          </li>
-          <li>Let the <a>[[\KeyingMaterial]]</a> internal slot of <var>output</var>
-          refer to the same private keying material represented by the
-          <a>[[\KeyingMaterial]]</a> internal slot of <var>input</var>.
-          </li>
+          <li>Set <var>serialized</var>.[[\Expires]] to the value of
+          <var>value</var>'s <code>expires</code> attribute.</li>
+
+          <li>Set <var>serialized</var>.[[\Certificate]] to
+          a copy of the unstructured binary data in
+          <var>value</var>'s <a>[[\Certificate]]</a> slot.</li>
+
+          <li>Set <var>serialized</var>.[[\KeyingMaterial]] to a serialization
+          of the private keying material represented by
+          <var>value</var>'s <a>[[\KeyingMaterial]]</a> slot.</li>
+        </ol>
+
+        <p>Their <a>deserialization steps</a>, given <var>serialized</var> and
+        <var>value</var>, are:</p>
+
+        <ol>
+          <li>Initialize <var>value</var>'s <code>expires</code> attribute to
+          contain <var>serialized</var>.[[\Expires]].</li>
```

Comment:
nit: Should we link these slots as well?

Reply:
I don't think that is necessary - the definition is like 10 lines before. WDYT?